letter-sender/src/main/java/com/christmas/letter/sender/service/LetterSenderEventPublisher.java

```diff
@@ -0,0 +1,24 @@
+package com.christmas.letter.sender.service;
+
+import com.christmas.letter.sender.configuration.AWSSNSConfig;
+import com.christmas.letter.sender.dto.LetterEvent;
+import io.awspring.cloud.sns.core.SnsTemplate;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class LetterSenderEventPublisher {
```

Comment:
Let's move this publisher and the LetterEvent to a new package, event for example, or listener or whatever makes you think of events :D